src/converters.ts

```diff
@@ -0,0 +1,109 @@
+import {
+  EntityFromIntegration,
+  RelationshipFromIntegration
+} from "@jupiterone/jupiter-managed-integration-sdk";
+import { Agent, WazuhManager } from "./provider";
+
+export const ACCOUNT_ENTITY_TYPE = "provider_account";
+export const ACCOUNT_ENTITY_CLASS = "Account";
+
+export const WAZUH_MANAGER_ENTITY_TYPE = "provider_wazuhmanager";
+export const WAZUH_MANAGER_ENTITY_CLASS = "Wazuhmanager";
+
+export const AGENT_ENTITY_TYPE = "provider_agent";
+export const AGENT_ENTITY_CLASS = "Agent";
```

Comment:
The data model specifies a [`HostAgent` class](http://docs.jupiterone.io/en/latest/@lifeomic/jupiter-data-model/docs/HostAgent.html).

```suggestion
export const AGENT_ENTITY_CLASS = "HostAgent";
```